mtags/src/main/scala-2/scala/meta/internal/pc/Keywords.scala

```diff
@@ -2,6 +2,9 @@ package scala.meta.internal.pc
 
 import scala.tools.nsc.reporters.StoreReporter
 
+import scala.meta.XtensionTokenizeInputLike
```

Comment:
```suggestion
```
This should not be needed. You might do `import scala.meta._` instead (if there are conflicts you can import inside the def)